 * Serving Flask app "api" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
Using TensorFlow backend.
Traceback (most recent call last):
  File "api.py", line 127, in <module>
    app.run(host="10.142.0.13", port=8000)
  File "/home/quangvy2703/env/lib/python3.6/site-packages/flask/app.py", line 990, in run
    run_simple(host, port, self, **options)
  File "/home/quangvy2703/env/lib/python3.6/site-packages/werkzeug/serving.py", line 1012, in run_simple
    inner()
  File "/home/quangvy2703/env/lib/python3.6/site-packages/werkzeug/serving.py", line 965, in inner
    fd=fd,
  File "/home/quangvy2703/env/lib/python3.6/site-packages/werkzeug/serving.py", line 808, in make_server
    host, port, app, request_handler, passthrough_errors, ssl_context, fd=fd
  File "/home/quangvy2703/env/lib/python3.6/site-packages/werkzeug/serving.py", line 701, in __init__
    HTTPServer.__init__(self, server_address, handler)
  File "/usr/lib/python3.6/socketserver.py", line 456, in __init__
    self.server_bind()
  File "/usr/lib/python3.6/http/server.py", line 136, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "/usr/lib/python3.6/socketserver.py", line 470, in server_bind
    self.socket.bind(self.server_address)
OSError: [Errno 98] Address already in use
